src/screens/AuthenticatorList/AuthenticatorListScreen.tsx

```diff
@@ -50,7 +50,6 @@ class AuthenticatorListScreen extends Component<Props, State> {
     const { navigation } = this.props;
     navigation.navigate(Route.ScanQrCode, {
       onBarCodeScan: (psbt: string) => {
-        navigation.goBack();
```

Comment:
Not sure if it was on purpose but since we invoke `goBack()` method in ScanQrCodeScreen.tsx method then this one is a repeat. At a result, the user was redirected not to one page back but to two screens back (Wallets dashboard in most cases).

I can revert it if needed